src/ops/vaccel_ops.h

```diff
@@ -4,19 +4,23 @@
 #include <stddef.h>
 #include <stdint.h>
 
-#define VACCEL_NO_OP        0
-#define VACCEL_BLAS_SGEMM   1
-#define VACCEL_IMG_CLASS    2
-#define VACCEL_IMG_DETEC    3
-#define VACCEL_IMG_SEGME    4
-#define VACCEL_FUNCTIONS_NR 5
+enum vaccel_op_type {
+	VACCEL_NO_OP = 0,
+	VACCEL_BLAS_SGEMM, 	/* 1 */
+	VACCEL_IMG_CLASS, 	/* 2 */
+	VACCEL_IMG_DETEC,    	/* 3 */
+	VACCEL_IMG_SEGME,    	/* 4 */
+	VACCEL_EXEC,		/* 5 */
+	VACCEL_FUNCTIONS_NR
+};
 
 static const char *vaccel_op_name[] = {
 	"noop",
 	"sgemm",
 	"image-classification",
 	"image-detection",
 	"image-segmentation",
+	"exec",
 };
 
 static inline const char *vaccel_op_type_str(uint8_t op_type)
```

Comment:
Sould update here the op_type to the new enum type

Reply:
ack